requirements_test.txt

```diff
@@ -52,3 +52,4 @@ types-pytz==2025.1.0.20250204
 types-PyYAML==6.0.12.20241230
 types-requests==2.31.0.3
 types-xmltodict==0.13.0.3
+types-aiobotocore[s3]==2.13.1
```

Comment:
I guess it's this change.

Reply:
Can we add this in a separate PR? And please keep the sorting :abcd:.

Reply:
👌🏻